server/cabal.project.dev-sh.local

```diff
@@ -9,6 +9,9 @@
 
 package *
   documentation: true
+  -- build with DWARF support. Still fairly useless, but see: 
+  --   https://www.haskell.org/ghc/blog/20200403-dwarf-1.html
+  debug-info: 2
```

Comment:
If it's fairly useless, shall we leave it out for now? With the intention of keeping the build plan as simple as possible.

Reply:
Good point. I would like to leave it in, and added a better comment to justify why. Let me know if you strongly object.